tests/config/test_storages.py

```diff
@@ -23,41 +25,38 @@
 from moto import mock_s3
 
 from nucypher.characters.lawful import Ursula
-from nucypher.config.storages import S3NodeStorage, InMemoryNodeStorage, TemporaryFileBasedNodeStorage, NodeStorage
+from nucypher.config.storages import (
+    S3NodeStorage,
+    InMemoryNodeStorage,
+    TemporaryFileBasedNodeStorage,
+    NodeStorage,
+    LocalFileBasedNodeStorage
+)
 
 MOCK_S3_BUCKET_NAME = 'mock-seednodes'
 S3_DOMAIN_NAME = 's3.amazonaws.com'
 
 
-@pytest.fixture(scope='function')
-def light_ursula(temp_dir_path):
-    db_name = 'ursula-{}.db'.format(10151)
-
-    node = Ursula(rest_host='127.0.0.1',
-                  rest_port=10151,
-                  db_filepath=db_name,  # TODO: Needs cleanup
-                  db_name=db_name,
-                  federated_only=True)
-    yield node
-    with contextlib.suppress(Exception):
-        os.remove(db_name)
-
+class BaseTestNodeStorageBackends:
 
-@pytest.fixture(scope='function')
-def memory_node_storage():
-    _node_storage = InMemoryNodeStorage(character_class=Ursula, federated_only=True)
-    _node_storage.initialize()
-    return _node_storage
+    @pytest.fixture(scope='class')
+    def light_ursula(temp_dir_path):
+        db_name = 'ursula-{}.db'.format(10151)
```

Comment:
Yeah - a port number we dont typically use to avoid collisions.

Reply:
Do we actually bind to this port _for real_?
In such case, I'd much prefer to pick an available port rather than asserting that this one is unused. Something like:
```
import socket
from contextlib import closing

def find_free_port():
    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
        s.bind(('', 0))
        return s.getsockname()[1]
```

Reply:
Yeah - nice.  This is a nice project-wide utility.

Reply:
A few things come to mind. 

- We dont want to interfere with the users existing services, or run tests on an unavailable port.  
- Perhaps we find a better way to mock ursula networking in tests in general (performance!).
- Let's escalate to an Issue

Reply:
All right, can merge this and make an issue!

Reply:
:heart_eyes: